uspace/lib/wndmgt/include/wndmgt.h

```diff
@@ -47,6 +47,7 @@ extern errno_t wndmgt_get_window_info(wndmgt_t *, sysarg_t,
     wndmgt_window_info_t **);
 extern void wndmgt_free_window_info(wndmgt_window_info_t *);
 extern errno_t wndmgt_activate_window(wndmgt_t *, sysarg_t, sysarg_t);
+extern errno_t wndmgt_deactivate_window(wndmgt_t *, sysarg_t, sysarg_t);
```

Comment:
For this new operation unit tests should be added to /data/helenos/master/uspace/lib/wndmgt/test/wndmgt.c, similar to PCUT_TEST(activate_window_failure) and PCUT_TEST(activate_window_success).